tests/engine/adhoc_test.go

```diff
@@ -16,18 +15,22 @@ import (
 	publicwasm "github.com/tetratelabs/wazero/wasm"
 )
 
-// ctx is a default context used to avoid lint warnings even though these tests don't use any context data.
-var ctx = context.Background()
-
-func TestJIT(t *testing.T) {
+func requireJITSupported(t *testing.T) {
 	if runtime.GOARCH != "amd64" && runtime.GOARCH != "arm64" {
```

Comment:
how about `wazero.JITSupported` instead of repeating it?

```diff
diff --git a/config_supported.go b/config_supported.go
index 69e3553..13ae5d2 100644
--- a/config_supported.go
+++ b/config_supported.go
@@ -2,6 +2,8 @@
 
 package wazero
 
+const JITSupported = true
+
 // NewRuntimeConfig returns NewRuntimeConfigJIT
 func NewRuntimeConfig() *RuntimeConfig {
        return NewRuntimeConfigJIT()
diff --git a/config_unsupported.go b/config_unsupported.go
index 435fb38..c247b7d 100644
--- a/config_unsupported.go
+++ b/config_unsupported.go
@@ -2,6 +2,8 @@
 
 package wazero
 
+const JITSupported = false
+
 // NewRuntimeConfig returns NewRuntimeConfigInterpreter
 func NewRuntimeConfig() *RuntimeConfig {
        return NewRuntimeConfigInterpreter()
```

Reply:
love it !